Makefile

```diff
@@ -69,6 +70,8 @@ make_ldflags = $(2) -X $(LND_PKG)/build.Commit=lightning-terminal-$(COMMIT) \
 	-X $(LND_PKG)/build.RawTags=$(shell echo $(1) | sed -e 's/ /,/g') \
```

Comment:
Note that we do not change the:
`$(LND_PKG)/build.Commit=lightning-terminal-$(COMMIT)`

above here, so that the `Commit` in response through `lncli` remains the same. I think it doesn't make sense to change this until we also change the output of `getinfo` in `lnd`.